src/main/groovy/nextflow/cli/CmdClean.groovy

```diff
@@ -66,6 +66,9 @@ class CmdClean extends CmdBase implements CacheBase {
     @Parameter(names='-but', description = 'Clean up all runs except the specified one')
     String but
 
+    @Parameter(names='-keepLogs', description = 'Removes only temp files but keeps the "dot" files')
+    boolean keepLogs
```

Comment:
For consistency I would propose to rename it to `-keep-logs`, maybe adding a `-k` shortcut. The label would be better as `Removes only temporary files but keeps execution logs and metadata`